.gemini/config.yaml

```diff
@@ -0,0 +1,10 @@
+have_fun: false # We are serious people in the Dart team!
```

Comment:
Is there a public link describing the file format + config options that we could reference from this file?